Sources/Purchasing/NonSubscriptionTransaction.swift

```diff
@@ -32,7 +32,7 @@ public final class NonSubscriptionTransaction: NSObject {
     @objc public let transactionIdentifier: String
 
     /// The unique identifier for the transaction created by the Store.
-    @objc internal let storeTransactionIdentifier: String
+    @objc public let storeTransactionIdentifier: String
```

Comment:
I'm only slightly concerned about the confussion of having both ids... but I think it shoud be ok?